packs/mssql/actions/execute.query_and_email.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: "execute.query_and_email"
+description: "Executes an MS SQL Query and emails results as attachments"
+enabled: true
+
+runner_type: "action-chain"
+entry_point: "workflows/execute.query_and_email.yaml"
+
+parameters:
+  query_string:
```

Comment:
Please ignore the comment above.

I some how missed `params` parameter in the metadata file which is defined just below (lol).